src/modules/esl-carousel/core/esl-carousel.renderer.ts

```diff
@@ -123,10 +123,18 @@ export abstract class ESLCarouselRenderer implements ESLCarouselConfig {
   /** Post-processing animation action. */
   public async onAfterAnimate(index: number, direction: ESLCarouselDirection): Promise<void> {}
 
-  /** Handles the slides transition. */
-  public abstract onMove(offset: number): void;
-  /** Ends current transition and makes permanent all changes performed in the transition. */
-  public abstract commit(offset?: number): void;
+  /**
+   * Move slide by the passed offset in px.
```

Comment:
just to follow consistency
```suggestion
   * Moves slide by the passed offset in px.
```